<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1231</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_hint">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите фамилию сотрудника</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_input">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_search">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Искать</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_Main">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>721</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_changeinfo">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>530</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Изменить</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_info">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Чтобы получить весь список нажмите сразу кнопку Искать</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>60</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Jobs</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>FamilySituation</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Study</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>10</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Таблицы с ID</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
